.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,114 +1,133 @@
 name: "🕷️ Bug report"
 description: Report errors or unexpected behavior
-labels:
-- Issue-Bug
-- Needs-Triage
+labels: [Needs-Triage, Issue-Bug]
+
 body:
-- type: markdown
-  attributes:
-    value: Please make sure to [search for existing issues](https://github.com/microsoft/PowerToys/issues) before filing a new one!
-- type: input
-  attributes:
-    label: Microsoft PowerToys version
-    placeholder: 0.70.0
-    description: Hover over system tray icon or look at Settings
-  validations:
-    required: true
+  - type: checkboxes
```

Comment:
I don't agree with this. I don't like it when others do it and i honestly doubt it will change any behavior.  With some of the things getting tested out, i bet we we'll improve alerting for duplicates as well based on results from other repos.

Reply:
Yes the bot is nice. But a waste of time if we can **prevent** duplicates.